modules/compliance-filtering-failed-results.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * security/compliance_operator/compliance-operator-remediation.adoc
+
+[id="filtering-failed-compliance-check-results_{context}"]
+= Filtering failed compliance check results
+
+By default, the `ComplianceCheckResult` objects are labeled with several useful labels that allow you to query the checks and decide on the next steps after the results are generated.
+
+
+To list checks that belong to a specific suite, run:
+
+[source,terminal]
+----
+$ oc get compliancecheckresults -l compliance.openshift.io/suite=example-compliancesuite
+----
+
+To list checks that belong to a specific scan, run:
+
+[source,terminal]
+----
+$ oc get compliancecheckresults -l compliance.openshift.io/scan=example-compliancescan
+----
+
+Not all `ComplianceCheckResult` objects create `ComplianceRemediation` objects. Only `ComplianceCheckResult` objects that can be remediated automatically do. A
+`ComplianceCheckResult` object has a related remediation if it is labeled with the `compliance.openshift.io/automated-remediation` label. The name of the remediation is the same as the name of the check.
+
+To list all failing checks that can be remediated automatically, run:
+
+[source,terminal]
+----
+$ oc get compliancecheckresults -l 'compliance.openshift.io/check-status in (FAIL),compliance.openshift.io/automated-remmediation
+----
```

Comment:
Correct command would be :
`$ oc get compliancecheckresults -l 'compliance.openshift.io/check-status=FAIL,compliance.openshift.io/automated-remmediation`